commands/guild/twitch-announcer.js

```diff
@@ -0,0 +1,443 @@
+const { Command } = require('discord.js-commando');
+const { MessageEmbed } = require('discord.js');
+const TwitchStatusCommand = require('../other/twitchstatus');
+const db = require('quick.db');
+const {
+  twitchClientID,
+  twitchClientSecret,
+  prefix
+} = require('../../config.json');
+
+if (twitchClientID == null || twitchClientSecret == null)
+  return console.log(
+    `INFO: Twitch Commands removed from the list.\nMake sure you have twitchClientID and twitchClientSecret in your config.json to use Twitch Features`
+  );
+module.exports = class TwitchAnnouncerCommand extends Command {
+  constructor(client) {
+    super(client, {
+      name: 'twitch-announcer',
+      memberName: 'twitch-announcer',
+      aliases: ['twitchannouncer', 'twitchannounce', 'ta'],
+      group: 'guild',
+      guildOnly: true,
+      userPermissions: ['ADMINISTRATOR'],
+      clientPermissions: ['MANAGE_MESSAGES', 'MENTION_EVERYONE'],
+      examples: [
+        '```' + `${prefix}twitch-announcer enable`,
+        `${prefix}twitch-announcer disable`,
+        `${prefix}ta check` + '```'
+      ],
+      description:
+        'Allows you to ***Enable***, ***Disable*** or ***Check*** the Twitch Announcer.',
+      args: [
+        {
+          key: 'textRaw',
+          prompt:
+            'Would you like to ***Enable***, ***Disable*** or ***Check*** the Twitch Announcer?',
+          type: 'string',
+          oneOf: ['enable', 'disable', 'check']
+        }
+      ]
+    });
+  }
+
+  async run(message, { textRaw }) {
+    var Twitch_DB = new db.table('Twitch_DB');
+    var textFiltered = textRaw.toLowerCase();
+
+    //Error Missing DB
+    if (Twitch_DB.get(`${message.guild.id}.twitchAnnouncer`) == undefined)
+      return message.reply(
+        `:no_entry: No Settings were found, Please run ${prefix}twitch-announcer-settings command first`
+      );
+    message.delete();
+
+    //Get Twitch Ready for Response Embeds
+    const scope = 'user:read:email';
+    let access_token; // Token is only valid for 24 Hours (needed to repeat this in Ticker Sections)
+    let streamInfo;
+    let gameInfo;
```

Comment:
The variable getInfo is assigned a value in lines 269 and 345 but is never read or used, what is its purpose?

Reply:
moved to there actual locations